modules/planning/open_space/trajectory_smoother/planning_block.cu

```diff
@@ -0,0 +1,116 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "planning_block.h"
+#include <iostream>
+
+namespace apollo {
+namespace planning {
+bool InitialCuda() {
+    int dev = 0;
```

Comment:
We also prefer the same coding style for .cu files though it's not checked by "apollo.sh lint".
Plz try "scripts/clang-format.sh /path/to/some.cu".